django/sierra/sierra/tests/test_routers.py

```diff
@@ -0,0 +1,124 @@
+"""
+Tests catalog-api database routers.
+
+The default Django DB ("default") should contain data for all apps
+except the "base" app. Base app models model Sierra data and should
+pull from the Sierra DB ("sierra"). No tables should be present in both
+databases.
+"""
+
+import pytest
+from model_mommy import mommy
+
+from django.db import connections, DatabaseError
+from django.apps import apps
+from django.conf import settings
+
+from sierra import routers
+from export import models as em
+from base import models as bm
+
+
+SIERRA_TEST_MODEL_CLASS = apps.get_app_config('base').models.values()[0]
+EXPORT_TEST_MODEL_CLASS = apps.get_app_config('export').models.values()[0]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('model_class,database,op,expected', [
+    (EXPORT_TEST_MODEL_CLASS, 'default', 'read', True),
+    (SIERRA_TEST_MODEL_CLASS, 'sierra', 'read', True),
+    (EXPORT_TEST_MODEL_CLASS, 'sierra', 'read', False),
+    (SIERRA_TEST_MODEL_CLASS, 'default', 'read', False),
+    (EXPORT_TEST_MODEL_CLASS, 'default', 'write', True),
+    (SIERRA_TEST_MODEL_CLASS, 'sierra', 'write', DatabaseError),
+    (SIERRA_TEST_MODEL_CLASS, 'default', 'write', DatabaseError),
+])
+def test_sierra_router_db_for_op(model_class, database, op, expected):
+    """
+    Test that SierraRouter.db_for_{op} (db_for_read or db_for_write)
+    gives the expected result for the given model_class and database
+    """
+    test_method = getattr(routers.SierraRouter, 'db_for_{}'.format(op))
+    router, model = routers.SierraRouter(), model_class()
+    try:
+        result = test_method(router, model) == database
+    except Exception as e:
+        result = type(e)
+    assert result == expected
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('model_class,database,testing,expected', [
+    (EXPORT_TEST_MODEL_CLASS, 'default', False, True),
+    (EXPORT_TEST_MODEL_CLASS, 'default', True, True),
+    (EXPORT_TEST_MODEL_CLASS, 'sierra', False, False),
+    (EXPORT_TEST_MODEL_CLASS, 'sierra', True, False),
+    (SIERRA_TEST_MODEL_CLASS, 'sierra', False, False),
+    (SIERRA_TEST_MODEL_CLASS, 'sierra', True, True),
+    (SIERRA_TEST_MODEL_CLASS, 'default', False, False),
+    (SIERRA_TEST_MODEL_CLASS, 'default', True, False),
+])
+def test_sierra_router_allow_migrate(model_class, database, testing, expected):
+    """
+    Test that SierraRouter.allow_migrate returns the expected result
+    for the given model_class and database, depending on whether or not
+    it's a testing environment
+    """
+    router, model = routers.SierraRouter(), model_class()
+    settings.TESTING = testing
+    result = router.allow_migrate(database, model)
+    settings.TESTING = True
+    assert result == expected
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('model_class,database,expected', [
```

Comment:
So I did the exact same thing as you, not separating the params in the string with spaces, but it looks much nicer if you do add that separation, and it still works just fine.
`'model_class, database, expected'` vs `'model_class,database,expected'`